hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -458,33 +458,30 @@ def _get_default_analytic_line(self, date, project, task, timesheet=None):
                 'sheet_id': self.id,
             })
         if timesheet:
-            amount = sum([t.unit_amount for t in timesheet])
+            unit_amount = sum([t.unit_amount for t in timesheet])
             values.update({
-                'unit_amount': amount,
+                'unit_amount': unit_amount,
             })
         return values
 
     @api.model
-    def _prepare_empty_analytic_line(self):
+    def _prepare_empty_analytic_line(self, project, task):
         return {
             'name': '/',
```

Comment:
let's extract this constant somewhere

Reply:
?

Reply:
`'/'` is used in a lot of places, just an idea to make it a constant